### ISSUE TITLE
Submission text auto-format

### ISSUE THREAD

**Issue by ryscher** [datadryad/dryad-product-roadmap#3548]
GREI Y3 1.2

Auto-format free-text fields: Abstract, Methods to match (remove author's formatting that carries over via copy/paste). -- Make an "auto format" button that cleans some formatting, but not all.

Consult with curators for details.

**Comment by ahamelers** [comment]
@ryscher We already have a function that removes all the requested formatting ON paste. I've just checked this, and it is working as desired with text pasted from google docs, pdfs, and many random websites. No content pasted in has any formatting that should or would be removed by such a feature.

It seems superfluous to add an additional button to remove non-existent formatting on demand. No undesired formatting should be making it through the paste function to later be removed. If there is formatting that is making it through, that should be a bug ticket, or a feature request if it's something _really_ odd.

There are some descriptions that were created before we added this paste functionality that still have weird pasted formatting, and I suppose this feature could have some utility with those (currently, the solution is to copy and then paste the content in again, and I suppose a button would be faster?), but it would be extremely limited as eventually we'll run out of those
